services/web-app/components/status-indicator-table/status-indicator-row.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { StructuredListCell, StructuredListRow } from '@carbon/react'
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import {
+  cell,
+  descriptionCell,
+  glyph,
+  icon,
+  iconGroup,
+  iconWrapper,
+  isDark,
+  statusIndicatorRow
+} from './status-indicator.module.scss'
```

Comment:
```suggestion
import styles from './status-indicator.module.scss'
```

Reply:
then update the file below to be styles. for all the class names

Reply:
Done!